server/server.go

```diff
@@ -1546,7 +1550,7 @@ func PrintAndDie(msg string) {
 
 // PrintServerAndExit will print our version and exit.
 func PrintServerAndExit() {
-	fmt.Printf("nats-server: v%s\n", VERSION)
+	fmt.Printf("nats-server: v%s\n", serverVersion)
```

Comment:
What does serverVersion / VERSION contain for builds that are not a tag? Would a `git describe -tags` be an option, or is that's already what's used?

Reply:
`VERSION` is in the code: https://github.com/nats-io/nats-server/blob/main/server/const.go#L47

Reply:
So the way this would work now is that the dev version would show unless the build overwrites it with a tag

Reply:
I think for now maybe we could just include the changes to goreleaser yaml so that the proper metadata is included for the scanners and leave out changing VERSION dynamically.